### ISSUE TITLE
Provide a way to expand headers without showing edit controls

### ISSUE THREAD

**Issue by danielo515** [200ok-ch/organice#797]
Before creating a new issue, please read the "How we work with issues" section in the documentation: https://organice.200ok.ch/documentation.html#how_we_work_with_issues

**Is your feature request related to a problem? Please describe.**
When browsing my org tasks I like to expand them to have an overview. However, the only way to expand a header is by selecting it, and when you select it it opens the edit controls. This adds a lot of noise and makes harder to expand effectively, because the next target is not directly below the header, but below the edit controls. Not to mention that I'm always scared of accidentally edit the header.

**Describe the solution you'd like**
I like a way to expand a header without entering the edit view. Maybe a dedicated button, editing only when clicking on the text, or some other thing that allows me to navigate in read only mode.

**Describe alternatives you've considered**
Nothing that I can think of

**Additional context**
Orgzly has a `+` button at the very right of each task, maybe that is a workable way?

**Comment by munen** [comment]
Hi @danielo515 

Thank you for opening an issue. However, it's not really an issue, it's a UX discussion. Let me still answer a couple of points:

1. You are not in edit mode when you tap a header. You will have to tap one of the icons first.
2. You can close the action icons by tapping the space in the very top between "File browser" and the "undo" icon.
3. Even if you accidentally make changes to a header, you can always use the undo/redo functionality

As for adding one more tap to actually make editing possible, no, we will not do that. It assumes that most people use organice to read their files. Such a big UX change would have to come from a vote of a big chunk of the community.

If you want, you can make a proposal to add a 'read only' kind of mode. If the other maintainers like it, you're free to make a PR. Would that work for you?

**Comment by munen** [comment]
Updated the title for accuracy and removed the shouting.

**Comment by danielo515** [comment]
> * You are not in edit mode when you tap a header. You will have to tap one of the icons first.

That is something I understand, but I was not sure how to correctly name the thing that was annoying me.

> * You can close the action icons by tapping the space in the very top between "File browser" and the "undo" icon.

But that is a lot of unnecessary work (literally doubles the amount of effort 😄 ) when navigating deeply nested trees. I have to either tap twice, first on the node and then in a very specific area far away from it, or deal with the annoyance of not being able to comfortable reach the next nested node.
This is specially annoying when the next child item gets out of the view due to the big size of the controls and the fact that the entire UI may shift up or down to make room for the edit controls that I don't want to use until I'm reading to change a node, that doesn't happen often (I usually add more nodes, but I rarely edit existing ones).

> As for adding one more tap to actually make editing possible, no, we will not do that.

I'm not asking to increase the number of tabs to edit per-se. I will be happy enough having a way to navigate my tree without having to constantly deal with the controls I don't need. Doesn't matter if it is an extra button in the UI that most people interested in editing can ignore, or maybe putting the node controls in a specific fixed area that I can easily ignore rather than embedding them in the node itself.

**Comment by munen** [comment]
I'm very sorry to hear that organice is creating "unnecessary work", "annoyances", "not being comfortable" and "especially annoying" - all within two sentences. You are very welcome to use organice if it serves you and to not use it if it does not.

**Comment by munen** [comment]
As for creating more button taps for everyone who wants to use organice for editing, that is a no go as described before. Assuming that everyone has the same workflow as you - in a project that's years old and has thousands of users - is probably not right.

If you come up with a way to reconcile your wishes with the existing UX, feel free to open a constructive dialogue. In the meantime, I see nothing actionable in this issue and neither does it conform with our issue guideline: https://organice.200ok.ch/documentation.html#how_we_work_with_issues

**Comment by danielo515** [comment]
> You are very welcome to use organice if it serves you and to not use it if it does not.

Yes, and that is what I will exactly do, in fact is what I do with most things. But being an open source project I thought I could offer some suggestions to improve the experience of the users that could share my workflow.

> As for creating more button taps for everyone who wants to use organice for editing, that is a no go as described before.

We are clearly not understanding each other, because that is not what I am proposing. To better ilustrate what I'm requesting, here is an screenshot of a tree structure that allows navigation without triggering any aditional UI:
<img width="474" alt="image" src="https://user-images.githubusercontent.com/2270425/170692465-3dae7128-ad9b-489d-9227-33101dbb8c2c.png">

Taping/clicking on the `+` `-` symbols at the very right will expand/collapse the node without showing the controls dialog, while allowing the normal current workflow if you tap anywhere else.

> I'm very sorry to hear that organice is creating "unnecessary work", "annoyances", "not being comfortable" and "especially annoying" - all within two sentences.

I'm sorry if my way of communicating my discomfort with the workflow didn't come out as constructive as I wanted it to be. I usually start any issue to any open source project by thanking the creator for the tool and saying how cool it is, but in this case, because of the direct and impersonal nature of the template I just filled it in, without the required human touch it requires and I apologise for that. I think having a web-app to work with my org-mode files is a fantastic thing to have and I am very thankful that you made it open source, and I really want to use it. I just wanted to ask (in a bit more polite way than how I ended doing) if a feature to target one particular workflow could be added.

Regards
